check point (#11810)
PostHog/posthog.com: contents/docs/privacy/ccpa-compliance.mdx

This guide explains what the CCPA is, what data must be protected and what your options are for CCPA-compliant analytics. 

<CalloutBox type="info" title="This guide is not legal advice">

It is up to you to ensure you're compliant with regulations. We strongly recommend reading the relevant regulations in full and seeking independent legal advice regarding your obligations. 

</CalloutBox>

## What is the CCPA?

The California Consumer Privacy Act of 2018 (CCPA) gives consumers control over the personal information that businesses collect about them:

* The right to know about the personal information a business collects about them and how it is used and shared.

> * The right to delete personal information collected from them (with some exceptions). [Learn about deleting personal information](#deleting-personal-information-in-posthog).

Reviewer: ```suggestion
* The right to delete personal information collected from them (with some exceptions). [Learn about deleting personal information](/docs/privacy/data-storage#data-deletion).
```

* Derive 50% or more of their annual revenue from selling California residents' personal information.

Personal information is protected under CCPA and is considered information that identifies or could be linked with a person.

In short, any information which is tied to a specific individual can be considered Personal Information, from their social security number or license plate number to photos, emails, URLs, IP addresses or even pseudonyms.

## What is the impact of CCPA on product analytics?

The CCPA requires businesses to give consumers certain information in a "notice at collection". This means that, when people sign up to use your product, you need to explain how you intend to use their data to improve the product for them. 

A notice at collection must list the categories of personal information businesses collect about consumers and the purposes for which they use the categories of information. The notice must also contain a link to the business's privacy policy, where consumers can get more details on your privacy practices. [Learn more about controlling what you collect](/docs/privacy/data-collection).

> The right to delete personal information is also covered by CCPA. You have 45 days to respond to a request from a user to delete any personal information stored about them. [Learn more about deleting personal information](#deleting-personal-information-in-posthog).

Reviewer: ```suggestion
The right to delete personal information is also covered by CCPA. You have 45 days to respond to a request from a user to delete any personal information stored about them. [Learn more about deleting personal information](/docs/privacy/data-storage#data-deletion).
```

## How to set PostHog up for CCPA compliance

> We recommend using [PostHog Cloud US](/us) for CCPA compliance, but you can self-host your analytics on your own infrastructure and maintain full control of the data – i.e. you decide how and where to host any personal information. It also means you have full control of the underlying database, making it possible for you to easily share any information stored about an individual or delete any personal data. 

Reviewer: ```suggestion
We recommend using [PostHog Cloud US](https://us.posthog.com/) for CCPA compliance, but you can self-host your analytics on your own infrastructure and maintain full control of the data – i.e. you decide how and where to host any personal information. It also means you have full control of the underlying database, making it possible for you to easily share any information stored about an individual or delete any personal data. 
```

### Step 1: Choose a hosting provider

> We recommend using [PostHog Cloud US](/us) for CCPA compliance. If self-hosting, the steps will depend on where you're hosting your data. 

Reviewer: ```suggestion
We recommend using [PostHog Cloud US](https://us.posthog.com/) for CCPA compliance. If self-hosting, the steps will depend on where you're hosting your data. 
```

We also strongly recommend that you limit access to PostHog and the infrastructure it is deployed on only to people who are authorized and need to access the data, including shared dashboard links. Although aggregate data in dashboards should not contain personal data, it may be possible for malicious users to infer personal data unless it is evaluated thoroughly via expert determination.

Finally, we advise caution when installing, building and enabling [apps](/docs/apps) for your PostHog instance. Apps are a great way to share and augment data from your instance with other systems, but it's essential to ensure you have the proper controls in place when sharing personal data outside of your self-hosted PostHog instance.

> ## Deleting personal information in PostHog
> 

Reviewer: ```suggestion
```
